.gitignore

```diff
@@ -5,3 +5,4 @@
 /config.cache
 /BUILD
 /INST
+.vscode
```

Comment:
Can `.vscode` really be in any directory of the source tree?  If it's only at the top-level then it's better to say `/.vscode` as then git only checks for it in the same directory as the `.gitignore` file it's listed in.  Checking one extra ignore entry per directory doesn't make much difference, but if many needlessly broad entries can.

Also, you've misspelling "ignoring" in the commit message.  And it's conventional to start the commit summary with a capital letter (see the advice in the dev guide on writing commit messages).

Reply:
.vscode folder, would always be in the root of the project (top-level), although there's something called  [Multi-root Workspaces](https://code.visualstudio.com/docs/editor/multi-root workspaces), where we would have multiple roots and maybe multiple setting files. 
I think we shouldn't worry about that and we can only ignore the folder in the root

Reply:
I agree that we shouldn't worry about more complex setups. People can figure those out using global gitignore as needed.

For commit messages the "subject" (the first line) should generally be of the form "Fix something", "Do something", so in this case it'd be better to use something like "Ignore .vscode directory" or "Ignore VSCode settings" (both with additional explanation in the "body" of the commit message if necessary to make it clear what's going on, although in this case the change is probably clear enough with a subject only).

Reply:
(The reason for doing it like that is that git generates commit messages automatically sometimes — for merges, for instance — and phrases them in that way.)